### PR TITLE
Add test for printing debug info

### PR DESCRIPTION
**Pull request by rijobro**
### Description
Sorry, it seems I forgot to add the unit test for printing debug info.

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [x] New tests added to cover the changes.
- [x] Integration tests passed locally by running `./runtests.sh --codeformat --coverage`.
- [x] Quick tests passed locally by running `./runtests.sh --quick`.
- [x] In-line docstrings updated.
- [x] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

**Comment by wyli** [comment]
I actually forgot to mention that please add `psutil ` to this doc https://github.com/Project-MONAI/MONAI/blob/master/docs/source/installation.md#installing-the-recommended-dependencies (similar to `tqdm` and `lmdb`)

**Comment by rijobro** [comment]
@wyli I've also updated the bug report to use the debug output, so I'll also need to print OS, python version and CUDA/cuDNN version. 

As such, please don't merge this straight away.

**Comment by wyli** [comment]
sure there's a 'convert to draft' button on this PR page

**Comment by rijobro** [comment]
@wyli thanks, done now.

File: monai/utils/module.py [hunk 1]
```diff
@@ -231,3 +232,17 @@ def has_option(obj, keywords: Union[str, Sequence[str]]) -> bool:
...
+    """
+    Try to load package and get version. Unload once finished. If not found, return `default`.
+    """
+    dep_ver = default
+    dep, has_dep = optional_import(dep_name)
+    if has_dep:
+        if hasattr(dep, "__version__"):
+            dep_ver = dep.__version__
+        del dep
+        del sys.modules[dep_name]
```
Comment (wyli) [inline monai/utils/module.py:247]: if `dep_name` is already imported before `get_package_version`, this might cause some surprises...

Comment (rijobro) [reply monai/utils/module.py]: good point, I added in a check to only unload if not already loaded. And added a test to that effect.

**Review by wyli — APPROVED** [review]
thanks, passed all tests and also locally verified

**Pull request merged**